tests/helper/helper.go

```diff
@@ -209,6 +209,11 @@ func WaitForHpaCreation(t *testing.T, kc *kubernetes.Clientset, name, namespace
 	return hpa, err
 }
 
+func KubernetesScaleDeployment(t *testing.T, resourceName string, desiredReplica int64, namespace string) {
+	_, err := ExecuteCommand(fmt.Sprintf("kubectl scale deployment %s --namespace %s --replicas=%d", resourceName, namespace, desiredReplica))
+	assert.NoErrorf(t, err, "cannot scale resource - %s", resourceName)
+}
```

Comment:
Maybe we should use the own kubeclient to scale this, I mean, we have other cases with kubeClient itself and it brings better feedback in case of failure. 
WDYT?

Reply:
@JorTurFer I understand what you are saying i'll try that in the code .